test/e2e/atlas/kubernetes_config_generate_test.go

```diff
@@ -47,6 +48,11 @@ import (
 )
 
 const targetNamespace = "importer-namespace"
+const latestOperatorVersion = "1.6.0"
```

Comment:
[q] Does this var need to be updated for each operator release?

Reply:
@andreaangiolillo I fixed that, but I moved this constant to `crds.go`. As well as I removed the automatic calculation of what version is the latest, as go maps do not preserve the order. Yes, this needs to be updated for each minor (y-stream) release.

Reply:
isn't there a way to automatically get this value? For instance, you can add a step in the evergreen task to check the latest operator release and add it to an env variable that you can access inside this test.